secretmanager/snippets/regional_samples/disable_regional_secret_delayed_destroy.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+
+import argparse
+
+# [START secretmanager_disable_regional_secret_delayed_destroy]
+
+# Import the Secret Manager client library.
+from google.cloud import secretmanager_v1
+
+
+def disable_regional_secret_delayed_destroy(
+    project_id: str, location_id: str, secret_id: str
+) -> secretmanager_v1.Secret:
+    """
+    Disable delayed destroy on an existing secret with a version destroy ttl.
```

Comment:
suggestion: please consider adding argument and return value explanation to the docstring